crates/bws/src/config.rs

```diff
@@ -48,13 +48,13 @@ pub(crate) const DIRECTORY: &str = ".bws";
 
 pub(crate) fn get_config_path(config_file: Option<&Path>, ensure_folder_exists: bool) -> PathBuf {
```

Comment:
I wonder if this should actually return an error which is propagated up to the CLI?

Reply:
I didn't want to change it too much in case the function was used in a lot of places but turns out it's only used in this file by functions that already return Result, so I've changed it now.